selene-vscode/src/util.ts

```diff
@@ -125,23 +127,23 @@ export async function downloadSelene(directory: string) {
     }
 }
 
-export async function getSelenePath(storagePath: string): Promise<string | undefined> {
+export async function getSelenePath(storagePath: vscode.Uri): Promise<vscode.Uri | undefined> {
     const settingPath = vscode.workspace.getConfiguration("selene").get<string | null>("selenePath")
     if (settingPath) {
-        return settingPath
+        return vscode.Uri.file(settingPath)
     }
 
-    const downloadPath = path.join(storagePath, getSeleneFilename())
+    const downloadPath = vscode.Uri.joinPath(storagePath, getSeleneFilename());

```

Comment:
Are these not prototypes, does `storagePath.joinPath` not work?

Reply:
Also, don't use semicolons to be consistent with the rest of the code. I need to remember to put Prettier on here at some point.

Reply:
Doesn't look like it - https://code.visualstudio.com/api/references/vscode-api#Uri.

Apologies for the semicolons, I tried to remove them where possible, but I guess it's just muscle memory 😛